UPGRADE_NOTES.md

```diff
@@ -1,4 +1,79 @@
-### Upgrading from 0.14 to 0.15
+## Upgrading from 0.16 to 0.17
+
+- Ensure you've upgraded to crystal 0.30.0
+- Upgrade Lucky CLI (homebrew)
+
+```
+brew update
+brew upgrade crystal-lang # Make sure you're up-to-date. Requires 0.30.0
+brew upgrade lucky
+```
+
+- Upgrade Lucky CLI (Linux)
+
+- Update `.crystal-version` to `0.30.0`
+
+> Remove the existing Lucky binary and follow the Linux
+> instructions in this section
+> https://luckyframework.org/guides/getting-started/installing#on-linux
+
+- Update versions in `shard.yml`
+  - Lucky should be `~> 0.17`
+- Run `shards update`
+
+### General updates
+- Rename: Action rendering method `text` to `plain_text`.
+- Update: use of `number_to_currency` now returns a String instead of writing to the view directly.
+- Delete: `config/static_file_handler.cr`. The `Lucky::StaticFileHandler` no longer has config settings.
+- Add: a new `Lucky::LogHandler` configure to `config/logger.cr`.
+```
+Lucky::LogHandler.configure do |settings|
+  # Skip logging static assets in development
+  if Lucky::Env.development?
+    settings.skip_if = ->(context : HTTP::Server::Context) {
+      context.request.method.downcase == "get" &&
+      context.request.resource.starts_with?(/\/css\/|\/js\/|\/assets\//)
+    }
+  end
+end
+```
+
+### Database updates
+- Add: a new `AppDatabase` class in `config/database.cr` that inherits from `Avram::Database`. (e.g. `class AppDatabase < Avram::Database`)
```

Comment:
I did this differently. I put it in `src/app_database.cr` as I believe lucky_cli does: https://github.com/luckyframework/lucky_cli/blob/1fd4d7d961523711736df37dd205ff971e2fc0c2/src/web_app_skeleton/src/app_database.cr